aries-site/src/pages/components/pageheader.mdx

```diff
@@ -0,0 +1,220 @@
+  height={{ min: 'small' }}
+  showResponsiveControls={[]}
+  template
+>
+  <PageHeaderIntroExample />
+</Example>
+
+## Anatomy
+
+<Example
+  caption="Diagram illustrating the four main content regions of a PageHeader."
+  plain
+>
+  <PageHeaderAnatomy />
+</Example>
+
+1. **Page Identifier:** This region is intended to provide the user with context on what the page is about.
+
+   - **a.** [Title](#title) (required): A short and descriptive title that uniquely identifies the page.
+   - **b.** [Subtitle](#subtitle) (optional): Concise contextual information that helps elaborate the purpose of the page.
+
+2. [Contextual Navigation](#reverse-anchor): A Reverse Anchor is required on all child pages as a means of navigating up the information hierarchy.
+3. [Page-Level Actions](#page-level-actions) (optional): This region is reserved for actions that affect the entire page or kick-off a task flow.
+
+## Responsive Behavior and Content Prioritization
+
+The main purpose of a page header is to identify a page and its context within an application. Therefore, the title and subtitle regions should receive screen priority when they come into contact with page-level actions.
+
+The region containing the title and subtitle should:
+
+- Flex to fill the available horizontal space on the page.
+- Have a minimum width of "medium" on `medium-xlarge` breakpoints and "small" on `xsmall-small` breakpoints to avoid excessive wrapping.
+
+Page-level actions should:
+
+- Have a width of "auto" by default.
+- Progressively collapse into an overflow menu as screen width becomes limited. This should follow guidance on [responsive behavior of actions](#responsive-behavior-of-actions).
+
+Resize your screen to see how the content regions adapt at various screen widths.
+
+<Example showResponsiveControls={[]} screenContainer plain width="100%">
+  <PageHeaderContentRegions background="background-front" />
+</Example>
+
+## Title
+
+A title is a required element of the page and should:
+
+- Always be an h1 with `size="small"`.
+- Uniquely and concisely identify the page. No two pages under the same parent should have the same name.
+- Follow title case capitalization (i.e., List of Clusters, My Account).
+- Wrap if too long for the available space. See [Responsive Behavior and Content Prioritization](#responsive-behavior-and-content-prioritization) for more guidance.
+- Be left-aligned with page content.
+
+## Subtitle
+
+A subtitle is an optional element that can provide additional contextual information about the contents of the page.
+
+A subtitle should:
+
+- Concisely elaborate what the page is on. This could be additional information such as ownership of an application, timestamps, or a name to represent by whom a piece of information was created.
+- Be located below the title and left-aligned with page content.
+- Wrap at the max-width for the paragraph's font-size in order to optimize the readability.
+
+<BestPracticeGroup>
```

Comment:
Some of the examples are showing the Subtitle with 18px, `medium` `font-size`. Can we update those to `large` please :)

Reply:
Can you point me to which ones? I'm not seeing that.

If you're referencing the ones with NameValueList, that is because we are using the default NameValueList styling. I would expect "large" paragraph size only for the standard string subtitle.

Reply:
The Do's and Don'ts under "Subtitle" and the examples where we show the Child Page, Top-Level Page, Page-Level Actions but that's probably due to what you mentioned earlier on adjusting the sizes inside of the interactive examples container.

Reply:
Yes when you're viewing it inline in the page, it's a scaled down version just for that view. If you expand the example to "fullscreen" you'll see it in its normal scaling.

Reply:
I noticed that, just wasn't sure if it didn't scale correctly somehow. Thanks! :)